CHANGELOG.txt

```diff
@@ -2,6 +2,8 @@
 
 - Issue #1960656 by juampynr: Give other modules the chance to hook in once addthis variable is available [#1960656]
 - Issue #2407083 by petterw03: Add support for medium sized icons (20x20) [#2407083]
+- Issue #2505439 by julka: update Twitter settings to use supported variables (addthis_config.passthrough.twitter.via and addthis_config.passthrough.twitter.text instead of addthis_config.templates.twitter)
+- Modified default addthis_toolbox icons from facebook,twitter to preferred_1,preferred_2,preferred_3,preferred_4,compact as this provides more functionality and the most shares.
```

Comment:
I added this update already to the CHANGELOG. I added a section with github merges. See here: http://cgit.drupalcode.org/addthis/commit/?id=c2073940a9e6383ee0a31bc378981cafc0999d55